recipe/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
   run:
     - python
     - numpy
+    - msgpack-python
+    - backports.lzma  # [py27]
```

Comment:
Also we don't have this for Windows yet. So for now, it may be best to change the selector to `# [py27 and unix]`.

xref: https://github.com/conda-forge/backports.lzma-feedstock/issues/1